package.json

```diff
@@ -25,7 +25,7 @@
     "@commitlint/cli": "^12.0.0",
     "@commitlint/config-conventional": "^12.0.0",
     "husky": "^6.0.0",
-    "lerna": "^4.0.0",
+    "lerna": "^3.0.0",
```

Comment:
we need to keep an eye on dependabot trying to update (and automerge) this. alternatively, put an exclusion in dependabot.yml

Reply:
please always use the package manager to make these changes, or the lockfile won't be updated